inc/deploygroup.class.php

```diff
@@ -114,7 +114,11 @@ public function defineTabs($options = [])
         $this->addDefaultFormTab($ong);
 
         $count = self::getMatchingItemsCount("PluginGlpiinventoryTaskjob");
-        $ong[$this->getType() . '$task'] = self::createTabEntry(_n('Associated task', 'Associated tasks', Session::getPluralNumber(), 'glpiinventory'), $count);
+        if ($_SESSION['glpishow_count_on_tabs']) {
+            $tabs[2] = self::createTabEntry(_n('Associated task', 'Associated tasks', Session::getPluralNumber(), 'glpiinventory'), $count);
+        } else {
+            $tabs[2] = _n('Associated task', 'Associated tasks', Session::getPluralNumber());
+        }
```

Comment:
```suggestion
        if ($_SESSION['glpishow_count_on_tabs']) {
            $count = self::getMatchingItemsCount("PluginGlpiinventoryTaskjob");
            $tabs[2] = self::createTabEntry(_n('Associated task', 'Associated tasks', Session::getPluralNumber(), 'glpiinventory'), $count);
        } else {
            $tabs[2] = _n('Associated task', 'Associated tasks', Session::getPluralNumber());
        }
```

Reply:
and remove `$count = self::getMatchingItemsCount("PluginGlpiinventoryTaskjob");` L116

There's no point in counting if the total is not displayed